wire-ios/Wire-iOS/Sources/Authentication/Event Handlers/Device Configuration/DeviceConfigurationEventHandler.swift

```diff
@@ -23,8 +23,6 @@ class DeviceConfigurationEventHandler: AuthenticationEventHandler {
     weak var statusProvider: AuthenticationStatusProvider?
 
     func handleEvent(currentStep: AuthenticationFlowStep, context: Void) -> [AuthenticationCoordinatorAction]? {
-        guard case .configureDevice = currentStep else { return nil }
```

Comment:
is it safe to remove it so it's not triggered in other unwanted cases?

Reply:
You mean handling it in either the configureDevice or deleteClient step?

Reply:
yes

Reply:
sure I can do it like that.